### ISSUE TITLE
Use an example curvenote paper as test content on all templates

### ISSUE THREAD

**Issue by stevejpurves** [curvenote/templates#12]
Instead of using the template author default content, we could use content from an example curvenote article and just leave it there. That means that all thumbnails would be generated with the same content.

We should create that paper so that we can interesting content on the first page. cross-references, links, citations, a figure with caption etc...

We've thinking of doing this because then is we use that content to test with, we'll be testing against something that reflects the curvenote output schema and will catch issues like #11 

This is fine for templates that we add, but when we get contributions in from others this might be difficult to follow?

Or a particular template might be better demonstrated with specific example content, which the generic one might not show off?

An alternative would be to allow the example content for template to be as is, derived from the example  given in the original template that we port, and instead we provide different content in a `testing` folder for the explicit purpose of testing manually and automatically.

I'm erring on doing the second.... 

cc @rowanc1

**Comment by rowanc1** [comment]
Yeah, I like the default content derived from the original template content and a testing folder with a few articles!
